src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -0,0 +1,143 @@
+                  "http://www.eastendbrewing.com/"),
+                new Listing("The Shiny Bean Coffee & Tea", 
+                  new MapLocation(40.496328, -79.944862), 
+                  4.9, 
+                  "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", 
+                  "https://theshinybean.com/"),
+                new Listing("Weisshouse", 
+                  new MapLocation(40.456684, -79.925499), 
+                  4.3, 
+                  "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", 
+                  "https://www.weisshouse.com/")
+              };
+
+  private void populateListOfBusinessesIntoLinkedList(Listing[] arrayOfBusiness) {
+    for(Listing business: sampleListingsOfBusinesses){
+      sampleListOfBusinesses.add(business);
+    }
+  }
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private void setUpSampleDatabase() {
+    String title = "business";
+    Entity taskEntity = new Entity("BigBusinesses");
+    String[] businesses = {"McDonalds","west elm","LA Fitness"};
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    for(String business: businesses){
+      taskEntity = new Entity("BigBusinesses");
+      taskEntity.setProperty(title, business);
+      datastore.put(taskEntity);
+    }
+    assertEquals(3, datastore.prepare(new Query("BigBusinesses")).countEntities());
+  }
+
+  // To setup a sample database and see if it is created properly 
+  @Test
+  public void testSampleDatabase() {
+    setUpSampleDatabase();
+  }
+
+  @Test
+  public void testEliminateBigBusinessesMethod() {
+    setUpSampleDatabase();
+    populateListOfBusinessesIntoLinkedList(sampleListingsOfBusinesses);
+
+    SmallCityService testSmallCityService = 
+                                  new SmallCityService(googlesOffice);
+    testSmallCityService.setAllBusinesses(sampleListOfBusinesses);
+
+    // These are the listings that should be in the list 
+    // after the Bigbusinesses are removed from the List.
+    // Currently to make sure this test runs proberly, I made it so that 
+    // the refrences of the appropriate Listings are added to a seperate array, 
+    // so when eliminateBigBusinesses method is called, I am sure that the bigBusinesses 
+    // are being removed from the list variable in the SmallCityService class.
+    expectedListOfBusinesses.add(sampleListOfBusinesses.get(3));
+    expectedListOfBusinesses.add(sampleListOfBusinesses.get(4));
+    expectedListOfBusinesses.add(sampleListOfBusinesses.get(5));
+    
+    testSmallCityService.parseThroughTheListOfBusinesses();
+    
+    Assert.assertEquals(expectedListOfBusinesses, testSmallCityService.getBusinesses());
```

Comment:
Looks like this line is also too long. It can be split like:

```
Assert.assertEquals(
    expectedListOfBusinesses, testSmallCityService.getBusinesses());
```

Reply:
fixed